packages/core/cms/faststore/content-types.json

```
@@ -271,6 +271,46 @@
                       "default": "Receive Tomorrow"
                     }
                   }
+                },
+                "tags": {
+                  "title": "Delivery Promise tags",
+                  "type": "object",
+                  "description": "The tag specified in this section will be displayed in product cards. Only one tag can be displayed.",
```

```suggestion
                  "description": "The tag specified in this section will be displayed in product cards. Only one can be displayed at a time (Delivery Option or Delivery Estimate)",
```
Do you think we should be more specific?